B/Birch_Standard/build_tarballs.jl

```diff
@@ -0,0 +1,71 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Birch_Standard"
+# Birch's version is `(git describe --long || echo) | sed -E 's/v([0-9]+)-([0-9]+)-g[0-9a-f]+/\1.\2/'`
+# (see https://github.com/JuliaPackaging/Yggdrasil/pull/2156#discussion_r528224861)
+version = v"1.75"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/lawmurray/Birch.git", "a533990b59b04ef4cd20f8d4fe91c4a220df1cfa"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+# add libraries for building birch on host
+apk add boost-dev yaml-dev
+
+# build and install driver on host system
+cd ${WORKSPACE}/srcdir/Birch/driver/
+mkdir -p ${WORKSPACE}/srcdir/build_host
+autoreconf -vi
+bb_target=${MACHTYPE} CXX=${HOSTCXX} CPPFLAGS=-I/usr/include LDFLAGS=-L/usr/lib ./configure --prefix=${WORKSPACE}/srcdir/build_host --build=${MACHTYPE} --host=${MACHTYPE}
```

Comment:
Where is the `bb_target` variable used?

Reply:
Hmm I don't remember when and where I copied these lines for compiling for the host system and how I modified them... Maybe it's not needed :smile: